test/e2e/benchmark/benchmark.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/v2/test/e2e/testnet"
+)
+
+type BenchmarkTest struct {
+	*testnet.Testnet
+	manifest *Manifest
+}
+
+func NewBenchmarkTest(name string, manifest *Manifest) (*BenchmarkTest, error) {
+	// create a new testnet
+	testNet, err := testnet.New(name, seed,
+		testnet.GetGrafanaInfoFromEnvVar(), manifest.ChainID,
+		manifest.GetGenesisModifiers()...)
+	if err != nil {
+		return nil, err
+	}
+
+	testNet.SetConsensusParams(manifest.GetConsensusParams())
+	return &BenchmarkTest{Testnet: testNet, manifest: manifest}, nil
+}
+
+// SetupNodes creates genesis nodes and tx clients based on the manifest.
+// There will be manifest.Validators validators and manifest.TxClients tx clients.
+// Each tx client connects to one validator. If TxClients are fewer than Validators, some validators will not have a tx client.
+func (b *BenchmarkTest) SetupNodes() error {
+	testnet.NoError("failed to create genesis nodes",
+		b.CreateGenesisNodes(b.manifest.Validators,
+			b.manifest.CelestiaAppVersion, b.manifest.SelfDelegation,
+			b.manifest.UpgradeHeight, b.manifest.ValidatorResource))
+
+	// obtain the GRPC endpoints of the validators
+	gRPCEndpoints, err := b.RemoteGRPCEndpoints()
+	testnet.NoError("failed to get validators GRPC endpoints", err)
+	log.Println("validators GRPC endpoints", gRPCEndpoints)
+
+	// create tx clients and point them to the validators
+	log.Println("Creating tx clients")
+
+	err = b.CreateTxClients(b.manifest.TxClientVersion,
+		b.manifest.BlobSequences,
+		b.manifest.BlobSizes,
+		b.manifest.TxClientsResource, gRPCEndpoints)
+	testnet.NoError("failed to create tx clients", err)
+
+	// start the testnet
```

Comment:
Do the following lines just setup the testnet or actually start it? I think just setup so
```suggestion
	// Set up the testnet
```

which IMO the comment isn't necessary given there is a print statement that describes it below

Reply:
Right, it sets up the testnet, fixed it in https://github.com/celestiaorg/celestia-app/pull/3395/commits/62a8781f9c4db99c45247724d076e6a5d4a4485e